lib/hiera/puppet_function.rb

```diff
@@ -60,8 +60,11 @@ def hiera_block2(scope, key, override, &default_block)
 
   def lookup(scope, key, default, has_default, override, &default_block)
     unless Puppet[:strict] == :off
-      Puppet.warn_once('deprecations', self.class.name,
-        "The function '#{self.class.name}' is deprecated in favor of using 'lookup'. See https://docs.puppet.com/puppet/#{Puppet.minor_version}/reference/deprecated_language.html")
+      #TRANSLATORS 'lookup' is a puppet function and should not be translated
+      message = _("The function '%{class_name}' is deprecated in favor of using 'lookup'.") % { class_name: self.class.name }
+      message += ' '+ _("See https://docs.puppet.com/puppet/%{minor_version}/reference/deprecated_language.html") %
```

Comment:
Is joining on space generally OK in i18n?  I recognize these are separate sentences.

Reply:
That's how we've been told to do it so far.

Reply:
Having a space between sentences might not be "correct" in every language (we know of), but I don't think there's a case where it's absolutely **wrong** instead of being superfluous.